### PR TITLE
Fix amp lightbox toggle controls

### PR DESCRIPTION
**Pull request by cathyxz**
Addresses 2 issues: 
1. Lightbox doesn't relisten click on re-open. 
2. `GestureRecognizers` consume all touch events and prevent default, hence the click for toggling controls never reaches `amp-lightbox-viewer`.

### CODE REVIEW

File: src/image-viewer.js [hunk 1]
```diff
@@ -325,18 +325,18 @@ export class ImageViewer {
...
         this.motion_.halt();
+        event.preventDefault();
       }
     });
 
-    this.gestures_ = Gestures.get(this.viewer_);
-
     // Zoomable.
     this.gestures_.onGesture(DoubletapRecognizer, e => {
+      event.preventDefault();
```
Comment (cvializ) [inline src/image-viewer.js]: I think you want `e.preventDefault`

Comment (cathyxz) [reply src/image-viewer.js]: No actually, the e that we get from `GestureRecognizers` isn't an event. It's an object that contains custom read-only data pertaining to each gesture. For example, the one emitted by the `DoubletapRecognizer` is: 

`    this.signalEmit({clientX: this.lastX_, clientY: this.lastY_}, null);`

**Comment by cathyxz** [comment]
Looking into whether we can either pass the event down, or refire the original event in the case that none of the registered gesture recognizers fire after a certain period of time. The fundamental issue here is basically the double tap, tap zoom, and pinch recognizers consume a single tap when they shouldn't.

**Comment by cathyxz** [comment]
Not worth the rebase since I killed `image-viewer.js`.

**Pull request closed without merging**